easybuild/easyconfigs/d/dask/dask-2.3.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,92 @@
+easyblock = 'PythonBundle'
+
+name = 'dask'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://dask.org/'
+description = """Dask natively scales Python. Dask provides advanced parallelism for analytics, enabling performance
+ at scale for the tools you love."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('PyYAML', '5.1'),
+    ('SciPy-bundle', '2019.03'),
+    ('bokeh', '1.3.4', versionsuffix),
+]
+
+use_pip = True
+
+exts_list = [
+    ('fsspec', '0.4.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/fsspec'],
```

Comment:
@branfosj You can avoid listing `source_urls` for every extension by using the following:

```python
exts_default_options = {'source_urls': [PYPY_SOURCE]}

exts_list = [
    ('fsspec', '0.4.1', {
        'checksums': ['e37d78590e7a9adb5b165c9c5d1b650f08017173b40eb1efbdc02d6bdb0c753b'],
    ),
    ...
```

Reply:
done